db/migrate/20150504185433_add_workflow_version_to_classification.rb

```diff
@@ -0,0 +1,7 @@
+class AddWorkflowVersionToClassification < ActiveRecord::Migration
+  def change
+    add_column :classifications, :workflow_version, :text
+    Classification.update_all("workflow_version = metadata->>'workflow_version'")
```

Comment:
I don't think it's worth doing. That field won't be created for new classifications, and since we can't do a bulk update to remove a JSONB field. It'll take days and days to finish that migration on the Production db.